bundle/compliance/lib/common/common.rego

```diff
@@ -77,6 +77,16 @@ duration_gte(duration, min_duration) {
 	true
 }
 
+# The function determines whether the given date occurs earlier than the specified duration from the current time.
```

Comment:
should it be later? just from looking at the code
```suggestion
# The function determines whether the given date occurs later than the specified duration from the current time.
```

This is why I commented regarding the naming here, we basically checking to see if the provided date is greater than the current time minus the duration in nanoseconds, which means it came after the duration, or more specifically, within the duration, no?

So I'm not missing anything, maybe we can use `is_within_duration` or `date_within_duration`, wdyt? 

sorry for nitpicking I just think it's important to have to keep the common function clear as possible

Reply:
`within the duration` feels more comprehensive.
I agree it's important as this function probably will have more uses.
The description still feels too complex, I'll try to make it clearer.